tasks/upgrades.ts

```diff
@@ -189,20 +189,21 @@ task(
       }
 
       if (contractName === "Periphery") {
-        // deployedArtifact -> change to Periphery_Implemenatation.sol
-        deployedArtifactImpl = path.join(
+        // Move the new implementation artifact from Periphery.json to Periphery_Implemenatation.json
+        const proxyArtifactPath = path.join(
```

Comment:
we're making a backup to the Periphery.json contract that contains the proxy logic and the periphery impl logic. If the ABI changes for the periphery, this means the Periphery.json changes too. I think this mistake can be noticed rn, checkout the swap function in Periphery_Implementation (uint margin delta) vs Periphery (int margin delta).
The question is, do we use this 'combined' ABI anywhere? If not, we better just remove it completly.